db/data/regions/DE.yml

```diff
@@ -20,12 +20,25 @@ languages:
 - de
 - en
 example_address:
-  address1: Willy-Brandt-Straße 1
+  address1: Willy-Brandt-Straße ⁠1
```

Comment:
Updated the example `address1` to include the word-joiner. 

Thank you [failing build](https://github.com/Shopify/worldwide/actions/runs/10358088832/job/28671491628?pr=273)! 🙏